bq-workers/gitlab-parser/main.py

```diff
@@ -55,15 +58,20 @@ def index():
             if "X-Gitlab-Event" in headers:
                 event = process_gitlab_event(headers, msg)
 
-        shared.insert_row_into_bigquery(event)
+        if metadata["object_kind"] == "deployment":
```

Comment:
Acredito que aqui valeria um `and` melhor que dois if's

Reply:
Então, ficou assim porque a condição é exclusiva. Podemos ter o seguinte cenário:

Evento de deployment e environment de QA

object_kind = deployment
metadata["environment"] = qa

Se juntar os IFs em um and, ele vai cair no ELSE e vai publicar o evento.

A regra é ... **só pode salvar o evento de deployment se o o environment for de PROD** (ou outro(s) environment(s) que podemos definir no futuro)

Faz sentido?

Reply:
Ou da pra fazer assim 
```
if metadata != "deployment" or env == "prd":
    print("grava")
```

Reply:
Extrair a validação para um método, porque agora está aceitando múltiplos environments. 

Valeu Travi.